diff --git a/skills-client-integration/skills-int-client-vue/package.json b/skills-client-integration/skills-int-client-vue/package.json
index 4d2aa22..1818739 100644
--- a/skills-client-integration/skills-int-client-vue/package.json
+++ b/skills-client-integration/skills-int-client-vue/package.json
@@ -6,7 +6,7 @@
     "scripts": {
         "serve": "vue-cli-service serve",
         "buildDepsAndServe": "cd ../../skills-client-js/ && npm run build && cd ../skills-client-vue/ && npm run build && cd ../skills-client-integration/skills-int-client-vue/ && npm run serve",
-        "licenseDepCheck": "license-checker --production --onlyAllow='MIT;ISC;BSD-3-Clause;Apache-2.0' --summary",
+        "licenseDepCheck": "license-checker --production --onlyAllow='MIT;ISC;BSD-2-Clause;BSD-3-Clause;Apache-2.0' --summary",
         "licenseHeaderCheck": "license-check-and-add check -f '../license-add/license-add-config.json'",
         "licenseHeaderAdd": "license-check-and-add add -f ../license-add/license-add-config.json",
         "licenseCheck": "npm run licenseHeaderCheck && npm run licenseDepCheck ",
diff --git a/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js b/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js
index ae3c24a..718b7a2 100644
--- a/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js
+++ b/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js
@@ -418,7 +418,7 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // now visit the "Report Skills" (external) page
@@ -446,7 +446,7 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // now visit the "Report Skills" (external) page
@@ -456,7 +456,7 @@ context('Vue Tests', () => {
             // switch back to the the client display
             cy.get('[data-cy=userDisplayLink]').click()
             // verify we are still on the Rank Overview page
-            cy.clientDisplay(true).contains('Rank Overview');
+            cy.clientDisplay(true).contains('My Rank');
 
             // click the back button and verify that we are still in the
             // client display (Subject page)
