diff --git a/skills-client-integration/skills-int-e2e-test/cypress/integration/angular.specs.js b/skills-client-integration/skills-int-e2e-test/cypress/integration/angular.specs.js
index 25c5305..bfcd5ab 100644
--- a/skills-client-integration/skills-int-e2e-test/cypress/integration/angular.specs.js
+++ b/skills-client-integration/skills-int-e2e-test/cypress/integration/angular.specs.js
@@ -347,7 +347,7 @@ context('Angular Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // click the back button and verify that we are still in the
@@ -373,7 +373,7 @@ context('Angular Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             // click the browser back button and verify that we are still in the
@@ -392,11 +392,11 @@ context('Angular Tests', () => {
 
             // navigate to Rank Overview via direct link
             cy.visit('/angular/showSkills?skillsClientDisplayPath=%2Frank');
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // reload and confirm we are still on Rank Overview page
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
         });
 
         it('back button after reload', () => {
@@ -414,11 +414,11 @@ context('Angular Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // click the browser back button and verify that we are still in the
             // client display (Subject page)
@@ -444,7 +444,7 @@ context('Angular Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             cy.get('[data-cy=skillsDisplayPath]').contains('Skills Display Path: [/subjects/subj0/rank]');
         });
diff --git a/skills-client-integration/skills-int-e2e-test/cypress/integration/js.specs.js b/skills-client-integration/skills-int-e2e-test/cypress/integration/js.specs.js
index c5ea0c3..46ef818 100644
--- a/skills-client-integration/skills-int-e2e-test/cypress/integration/js.specs.js
+++ b/skills-client-integration/skills-int-e2e-test/cypress/integration/js.specs.js
@@ -303,7 +303,7 @@ context("Native JS Tests", () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // now visit the "Report Skills" (external) page
@@ -329,7 +329,7 @@ context("Native JS Tests", () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // click the back button and verify that we are still in the
@@ -355,7 +355,7 @@ context("Native JS Tests", () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             // click the browser back button and verify that we are still in the
@@ -374,11 +374,11 @@ context("Native JS Tests", () => {
 
             // navigate to Rank Overview via direct link
             cy.visit('/native/clientDisplay.html?skillsClientDisplayPath=%2Frank#/showSkills');
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // reload and confirm we are still on Rank Overview page
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
         });
 
         it('back button after reload', () => {
@@ -396,11 +396,11 @@ context("Native JS Tests", () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // click the browser back button and verify that we are still in the
             // client display (Subject page)
@@ -426,7 +426,7 @@ context("Native JS Tests", () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             cy.get('[data-cy=skillsDisplayPath]').contains('Skills Display Path: [/subjects/subj0/rank]');
         });
diff --git a/skills-client-integration/skills-int-e2e-test/cypress/integration/react.specs.js b/skills-client-integration/skills-int-e2e-test/cypress/integration/react.specs.js
index 0183898..38ac585 100644
--- a/skills-client-integration/skills-int-e2e-test/cypress/integration/react.specs.js
+++ b/skills-client-integration/skills-int-e2e-test/cypress/integration/react.specs.js
@@ -273,7 +273,7 @@ context('React Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // click the back button and verify that we are still in the
@@ -299,7 +299,7 @@ context('React Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             // click the browser back button and verify that we are still in the
@@ -318,11 +318,11 @@ context('React Tests', () => {
 
             // navigate to Rank Overview via direct link
             cy.visit('/react/index.html?skillsClientDisplayPath=%2Frank#/showSkills');
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // reload and confirm we are still on Rank Overview page
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
         });
 
         it('back button after reload', () => {
@@ -340,11 +340,11 @@ context('React Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // click the browser back button and verify that we are still in the
             // client display (Subject page)
@@ -370,7 +370,7 @@ context('React Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             cy.get('[data-cy=skillsDisplayPath]').contains('Skills Display Path: [/subjects/subj0/rank]');
         });
diff --git a/skills-client-integration/skills-int-e2e-test/cypress/integration/react_v17.specs.js b/skills-client-integration/skills-int-e2e-test/cypress/integration/react_v17.specs.js
index 4faf89b..7f4bd58 100644
--- a/skills-client-integration/skills-int-e2e-test/cypress/integration/react_v17.specs.js
+++ b/skills-client-integration/skills-int-e2e-test/cypress/integration/react_v17.specs.js
@@ -275,7 +275,7 @@ context('React17 Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // click the back button and verify that we are still in the
@@ -301,7 +301,7 @@ context('React17 Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             // click the browser back button and verify that we are still in the
@@ -320,11 +320,11 @@ context('React17 Tests', () => {
 
             // navigate to Rank Overview via direct link
             cy.visit('/react/index.html?skillsClientDisplayPath=%2Frank#/showSkills');
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // reload and confirm we are still on Rank Overview page
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
         });
 
         it('back button after reload', () => {
@@ -342,11 +342,11 @@ context('React17 Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // click the browser back button and verify that we are still in the
             // client display (Subject page)
@@ -372,7 +372,7 @@ context('React17 Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             cy.get('[data-cy=skillsDisplayPath]').contains('Skills Display Path: [/subjects/subj0/rank]');
         });
diff --git a/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js b/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js
index 8c28bd8..caddc35 100644
--- a/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js
+++ b/skills-client-integration/skills-int-e2e-test/cypress/integration/vue.specs.js
@@ -420,7 +420,7 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // now visit the "Report Skills" (external) page
@@ -448,7 +448,7 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview that contains the back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('exist');
 
             // now visit the "Report Skills" (external) page
@@ -458,7 +458,7 @@ context('Vue Tests', () => {
             // switch back to the the client display
             cy.get('[data-cy=userDisplayLink]').click()
             // verify we are still on the Rank Overview page
-            cy.clientDisplay(true).contains('Rank Overview');
+            cy.clientDisplay(true).contains('My Rank');
 
             // click the back button and verify that we are still in the
             // client display (Subject page)
@@ -485,7 +485,7 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             // click the browser back button and verify that we are still in the
@@ -504,11 +504,11 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview via direct link
             cy.visit('/vuejs/?skillsClientDisplayPath=%2Frank#/showSkills?refreshPage=true');
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // reload and confirm we are still on Rank Overview page
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
         });
 
         it('back button after reload', () => {
@@ -526,11 +526,11 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
             cy.clientDisplay().find('[data-cy=back]').should('not.exist');
 
             cy.reload();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             // click the browser back button and verify that we are still in the
             // client display (Subject page)
@@ -556,7 +556,7 @@ context('Vue Tests', () => {
 
             // navigate to Rank Overview and that it does NOT contains the internal back button
             cy.clientDisplay().find('[data-cy=myRank]').click();
-            cy.clientDisplay().contains('Rank Overview');
+            cy.clientDisplay().contains('My Rank');
 
             cy.get('[data-cy=skillsDisplayPath]').contains('Skills Display Path: [/subjects/subj0/rank]');
         });
diff --git a/skills-client-integration/skills-int-e2e-test/package.json b/skills-client-integration/skills-int-e2e-test/package.json
index 9f4eb9f..201911f 100644
--- a/skills-client-integration/skills-int-e2e-test/package.json
+++ b/skills-client-integration/skills-int-e2e-test/package.json
@@ -31,8 +31,8 @@
     "backend:start:ci:background": "mkdir -p ./target/logs && npm run backend:start:ci | awk '/ClientLoggingController/{ print > \"./target/logs/skills-client.out\" }!/ClientLoggingController/{ print > \"./target/logs/skills-service.out\" }' &",
     "backend:start:ci:background:oauth": "mkdir -p ./target/logs && npm run backend:start:ci:oauth | awk '/ClientLoggingController/{ print > \"./target/logs/skills-client.out\" }!/ClientLoggingController/{ print > \"./target/logs/skills-service.out\" }' &",
     "backend:start:release": "java -Dspring.config.location=classpath:/application.yml,file:serverConfigs/backend_application.properties -jar ./target/servicesJars/skills-service-*.jar",
-    "backend:waitToStart": "npx wait-port -v -t 60000 localhost:8080",
-    "backend:resetDb": "java -cp ./db/h2-1.4.200.jar org.h2.tools.RunScript -url jdbc:h2:tcp://localhost:9090/mem:skills -user sa -script ./db/reset.sql",
+    "backend:waitToStart": "npx wait-port -t 60000 localhost:8080",
+    "backend:resetDb": "export PGPASSWORD=skillsPassword && psql -U postgres -h localhost -d skills -f ./db/reset.sql",
     "release": "release-it --dry-run",
     "cyServices:start:skills-service": "npm run backend:start:background && npm run backend:waitToStart && echo 'skill-service started!!'",
     "cyServices:start:skills-service:ci": "npm run backend:start:ci:background && npm run backend:waitToStart && echo 'skill-service started!!'",
